orcoursetrion/tests/test_github.py

```diff
@@ -466,6 +487,25 @@ def test_actions_rerun_xml_success(self, config):
 
     @mock.patch('orcoursetrion.actions.github.config')
     @httpretty.activate
+    def test_actions_release_XML_success(self, config):
+        """Test the API call comes through as expected.
+        """
+        config.ORC_GH_OAUTH2_TOKEN = self.OAUTH2_TOKEN
+        config.ORC_GH_API_URL = self.URL
+        config.ORC_COURSE_PREFIX = self.TEST_PREFIX
+        config.ORC_XML_ORG = self.ORG
+        config.ORC_PRODUCTION_GITRELOAD = self.TEST_PRODUCTION_GR
+
+        self.register_hook_create(json.dumps({'id': 2}), status=201)
+        self.register_team_repo_add(self.callback_team_repo)
```

Comment:
Similar to the above comments about release_studio and release_xml, should the above be factored out into a common path?